test/gtest/ucp/test_ucp_dt.cc

```diff
@@ -181,6 +181,39 @@ class test_ucp_dt_iter : public ucs::test_with_param<ucp_datatype_t> {
 
 ucp_dt_generic_t* test_ucp_dt_iter::dt_gen = 0;
 
+/* The following tests reuse the generic datatype and its packed size will be
+ * set after the first packing. */
+UCS_TEST_P(test_ucp_dt_iter, datatype_query) {
```

Comment:
```suggestion
UCS_TEST_P(test_ucp_dt_iter, datatype_query) 
{
```

Reply:
The existing `UCS_TEST_P` have the curly block opening bracket on the same line. I am okay with changing, but I wanted to be consistent with the existing code.